### PR TITLE
Remove lfs

### PR DESCRIPTION
**Pull request by arnaudsj**
Hi Jhenna,

All the datasets except 1 should not be using LFS anymore, zipped them all, and updated the notebooks (hopefully did not miss any).

Please create a Google Drive Shared folder owned by Springboard when you have a second, share it with me so that I can add the only file that can't be compressed < 100 Mo (combined_data.csv in the recommendation mini project). and I can then update the pointer in the notebook and remove it from the repo as well.

Do not merge as is though, I want to make sure we get all the files in one sweep merge against the main branch.

Thanks!

### CODE REVIEW

**Comment by arnaudsj** [comment]
@jvoorhis-sb just added the Google drive download. Please review and merge if everything looks good!

File: mec-17.4.1-recommendation-systems-mini-project/Mini_Project_Recommendation_Systems.ipynb [hunk 1]
```diff
@@ -306,7 +306,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Dowload large file from Sprinboard's shared GDrive folder\n",
```
Comment (jvoorhis-sb) [inline mec-17.4.1-recommendation-systems-mini-project/Mini_Project_Recommendation_Systems.ipynb]: Could you take out the word Springboard here and just say "the shared GDrive"? That way it is transferable for UP versions.

**Review by jvoorhis-sb — CHANGES_REQUESTED** [review]
One tiny change in the notebook for 17.4.1

**Review by jvoorhis-sb — APPROVED** [review]
Looks good!

**Pull request merged**